src/Service/DashboardHelper.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Municipality;
+use App\Entity\User;
+use App\Repository\BoardRepository;
+use App\Repository\CaseEntityRepository;
+use App\Repository\UserRepository;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class DashboardHelper
+{
+    private BoardRepository $boardRepository;
+    private CaseEntityRepository $caseRepository;
+    private TranslatorInterface $translator;
+    private UrlGeneratorInterface $router;
+    private UserRepository $userRepository;
+
+    public function __construct(BoardRepository $boardRepository, CaseEntityRepository $caseRepository, TranslatorInterface $translator, UrlGeneratorInterface $router, UserRepository $userRepository)
+    {
+        $this->boardRepository = $boardRepository;
+        $this->caseRepository = $caseRepository;
+        $this->translator = $translator;
+        $this->router = $router;
+        $this->userRepository = $userRepository;
+    }
+
+    public function getDashboardGridInformation(Municipality $municipality, User $user): array
+    {
+        $gridInformation = [];
+
+        $gridInformation = $this->addUserColumnCaseInformation($user, $gridInformation);
+
+        $gridInformation = $this->addBoardsColumnCaseInformation($municipality, $gridInformation);
+
+        return $gridInformation;
+    }
+
+    private function addUserColumnCaseInformation(User $user, array $gridInformation): array
+    {
+        // Detect which filter option current user is
+        $userFilterOption = $this->getCurrentUserFilterOption($user);
+
+        // Construct the filter urls and do the counts
+        // In hearing
+        $hearingUrl = $this->router->generate('case_index', ['case_filter' => [
+            'assignedTo' => $userFilterOption,
```

Comment:
Why don't we use `$user->getId()` here? It does not seem right to use a numeric index when already have a unique id.

Reply:
Added `choice_value` field to both `assignedTo` and `board` filter in 0bc985fd241e5b099850ca3dc6e3109308eadbf4 to allow for use of `$user->getId()` and `$board->getId()` respectively.